appinventor/components/src/com/google/appinventor/components/runtime/ListAdapterWithRecyclerView.java

```diff
@@ -0,0 +1,347 @@
+
+        TextView textViewFirst=new TextView(context);
+        idFirst=View.generateViewId();
+        textViewFirst.setId(idFirst);
+        LinearLayout.LayoutParams layoutParams1=new LinearLayout.LayoutParams(LinearLayout.LayoutParams.WRAP_CONTENT,LinearLayout.LayoutParams.WRAP_CONTENT);
+        layoutParams1.topMargin = 10;
+        textViewFirst.setLayoutParams(layoutParams1);
+        textViewFirst.setTextSize(detailTextSize);
+        textViewFirst.setTextColor(detailTextColor);
+
+        LinearLayout linearLayout1= new LinearLayout(context);
+        LinearLayout.LayoutParams layoutParamslinear1 = new LinearLayout.LayoutParams(LinearLayout.LayoutParams.MATCH_PARENT,LinearLayout.LayoutParams.WRAP_CONTENT);
+        linearLayout1.setLayoutParams(layoutParamslinear1);
+        linearLayout1.setOrientation(LinearLayout.HORIZONTAL);
+        
+        linearLayout1.addView(textViewFirst);
+        cardView.setLayoutParams(params1);
+        cardView.addView(linearLayout1);
+        }else if(layoutType==Component.LISTVIEW_LAYOUT_IMAGE_SINGLE_TEXT){
+
+        TextView textViewFirst=new TextView(context);
+        idFirst=View.generateViewId();
+        textViewFirst.setId(idFirst);
+        LinearLayout.LayoutParams layoutParams1=new LinearLayout.LayoutParams(LinearLayout.LayoutParams.WRAP_CONTENT,LinearLayout.LayoutParams.WRAP_CONTENT);
+        layoutParams1.topMargin = 10;
+        textViewFirst.setLayoutParams(layoutParams1);
+        textViewFirst.setTextSize(detailTextSize);
+        textViewFirst.setTextColor(detailTextColor);
+
+        ImageView imageView = new ImageView(context);
+        idImages=View.generateViewId();
+        imageView.setId(idImages);
+        LinearLayout.LayoutParams layoutParamsImage = new LinearLayout.LayoutParams(imageWidth, imageHeight);
+        imageView.setLayoutParams(layoutParamsImage);
+        
+        LinearLayout linearLayout1= new LinearLayout(context);
+        LinearLayout.LayoutParams layoutParamslinear1 = new LinearLayout.LayoutParams(LinearLayout.LayoutParams.MATCH_PARENT,LinearLayout.LayoutParams.WRAP_CONTENT);
+        linearLayout1.setLayoutParams(layoutParamslinear1);
+        linearLayout1.setOrientation(LinearLayout.HORIZONTAL);
+        
+        linearLayout1.addView(imageView);
+        linearLayout1.addView(textViewFirst);
+        cardView.setLayoutParams(params1);
+        cardView.addView(linearLayout1);
+        }
+      else if(layoutType==Component.LISTVIEW_LAYOUT_IMAGE_TWO_TEXT){
+
+        TextView textViewFirst=new TextView(context);
+        idFirst=View.generateViewId();
+        textViewFirst.setId(idFirst);
+        LinearLayout.LayoutParams layoutParams1=new LinearLayout.LayoutParams(LinearLayout.LayoutParams.WRAP_CONTENT,LinearLayout.LayoutParams.WRAP_CONTENT);
+        layoutParams1.topMargin = 10;
+        textViewFirst.setLayoutParams(layoutParams1);
+        textViewFirst.setTextSize(detailTextSize);
+        textViewFirst.setTextColor(detailTextColor);
```

Comment:
You have TextSize/Color and DetailTextSize/Color parameters reversed. textViewFirst should use the TextSize/Color params, and testViewSecond should use DetailText.

The parameter names are displayed alphabetically, making it kind of confusing because they are placed far apart. I discussed this with Ankriti on her PR, and she agreed to change the names to TextMainSize/Color and TextDetailSize/Color so that they will sort together and make clearer which is which.